modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/portlet/action/GetSystemDataSetsMVCResourceCommand.java

```diff
@@ -81,7 +87,7 @@ protected void doServeResource(
 			JSONUtil.put(
 				"items",
 				JSONUtil.toJSONArray(
-					systemFDSNames,
+					systemFDSNamesSortedList,
```

Comment:
Can we simplify this to something like
```
JSONUtil.toJSONArray(
    new TreeSet<String>(systemFDSNames)
```
In any case, we can make the sort inline, and prevent variable naming issues.

Just like my import PR, this PR may need updating after Dani and Brian settle on naming. For example, it may no longer be `getSystemFDSNames`.